_data/sites.json

```diff
@@ -11978,6 +11978,17 @@
         ]
     },
 
+    {
+        "name": "OmaPosti",
+        "url": "https://asiakastiedot.posti.fi/myaccount/details",
+        "difficulty": "easy",
+        "notes": "In the bottom of the page, click 'Stop using Posti’s electronic services'.",
+        "notes_fi": "Sivun lopussa klikkaa 'Lopeta Postin sähköis­ten pal­ve­luiden käyttö'.",
```

Comment:
Finnish (?) is not a language we currently support, so we error out on the notes and they should be removed. If you want it to be supported, you can refer to https://github.com/jdm-contrib/jdm/blob/master/CONTRIBUTING.md#translation and follow the steps there :)
```suggestion
```